### PR TITLE
Aw 6 build the remaining landing page

### PR DESCRIPTION
**Pull request by JosephChuksN**
check if the file structure is good

### CODE REVIEW

**Comment by eimaam** [comment]
> check if the file structure is good

It's pretty fine! Now I get your suggestion of the Folder for Home components.
I actually created a pages folder instead.

Should we switch back to having only components folder?

**Comment by JosephChuksN** [comment]
> 

No! the pages folder is nice we can have them both.
pages folder can take other pages as well but the home folder contains components exclusively for the home page.

**Comment by eimaam** [comment]
> > 
> 
> No! the pages folder is nice we can have them both. pages folder can take other pages as well but the home folder contains components exclusively for the home page.

Okay! Great!
Will review and merge when you done! lets goo!

File: src/components/Home/Categories.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import shirt from './assets/shirt.jpg'
+import pant from './assets/pant.jpg'
+import shoes from './assets/shoe.jpg'
+import assesories from './assets/assesories.jpg'
+
+const Categories:React.FC = () => {
+  return (
+    <div className="px-5 font-sora pb-20">
+            <h1 className="text-base pb-3 font-semibold">Shop by categories</h1>
```
Comment (eimaam) [inline src/components/Home/Categories.tsx:10]: This header is having different size from the other like-headers... **Shirts**, **Trousers**....

**Review by eimaam — APPROVED** [review]

**Review by eimaam — APPROVED** [review]

**Comment by JosephChuksN** [comment]
I just checked deployment and i didn't find the other home components

**Comment by eimaam** [comment]
There was an issue with merge yesterday.
I've resolved it and merged.
You can pull from the main to see the structure.

**Pull request merged**